modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc

```diff
@@ -0,0 +1,749 @@
+= Deploy Redpanda Connectors in Kubernetes
+:description: Learn how to deploy and configure Redpanda Connectors using Helm. Redpanda Connectors is a package that includes Kafka Connect and built-in connectors that you can create and configure in Redpanda Console.
+:link-dockerhub: https://hub.docker.com/r/redpandadata/connectors/tags
+:page-context-links: [{"name": "Docker", "to": "deploy:deployment-option/self-hosted/docker-image.adoc" },{"name": "Kubernetes", "to": "deploy:deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc" } ]
+:page-toclevels: 1
+
+This topic describes how to use the Redpanda Helm chart to configure and deploy Redpanda Connectors in Kubernetes.
+
+Redpanda Connectors is a package that includes glossterm:Kafka Connect['Kafka Connect is a framework that provides a consistent and reliable platform to run and manage connectors.'] and the following built-in glossterm:connectors['Connectors are the plugins that interface with external systems. Connectors are plugged into Kafka Connect to facilitate data flow between Redpanda and these external systems.'], sometimes known as plugins. You can create and configure the built-in connectors in Redpanda Console.
+
+[cols="1a,2a"]
+|===
+|Built-In Connector|Description
+
+|MirrorSourceConnector
+|A glossterm:source connector['A source connector imports data from a source system into a Redpanda cluster. The source could be a cloud storage system such as S3 or GCS. The source connector's main task is to fetch data from these sources and convert them into a format suitable for Redpanda.'] that replicates records between multiple Kafka clusters. It is part of Kafka's MirrorMaker, which provides capabilities for mirroring data across Kafka clusters.
+
+|MirrorCheckpointConnector
+|A source connector that ensures the mirroring process can resume from where it left off in case of failures. It tracks and emits checkpoints that mirror the offsets of the source and target clusters.
+
+|MirrorHeartbeatConnector
+|A source connector that emits heartbeats to target topics at a regular interval, enabling MirrorMaker to track active topics on the source cluster and to synchronize consumer groups across clusters.
```

Comment:
Seems to introduce inconsistency with the Docker version. Why not make them consistent?